python/lsst/ip/isr/crosstalk.py

```diff
@@ -717,20 +720,31 @@ def run(self, exposure, crosstalk=None,
 
             if crosstalk.interChip:
                 if crosstalkSources:
+                    # Parse crosstalkSources: Identify which detectors we have available
+                    if isinstance(crosstalkSources[0], lsst.afw.image.Exposure):
+                        # Received afwImage.Exposure
+                        sourceNames = [exp.getDetector().getName() for exp in crosstalkSources]
+                    else:
```

Comment:
To be a little better at catching weird situations, I suggest making this an `elif` and having an `else` which raises.